app/helpers/conviction_helper.rb

```diff
@@ -0,0 +1,17 @@
+module ConvictionHelper
+  def conviction_title_string(object)
+    return t('conviction_title.never', scope: 'results/conviction') unless object.instance_of?(Date)
+
+    statement_type = object < Date.today ? 'past' : 'present'
+    statement = t("conviction_title.#{statement_type}", scope: 'results/conviction')
```

Comment:
We can pass the date here `date: I18n.l(object)` and then parametrise the locale (see comment below).

That way if we were to alter the order where the date appears in the sentence, (or even add a fullstop after the date) we don't need to touch the code here, just the locale.